src/screens/NoteListScreen.tsx

```diff
@@ -147,7 +147,7 @@ export default function NoteListScreen(props: PropsType) {
       <FAB
         icon="plus"
         accessibilityLabel="New"
-        color="white"
+        color={(theme.dark) ? "#000000" : "#ffffff"}
```

Comment:
I'm not going to complain because it was already shit before, but this should probably be in theme too. :P